apache/zeppelin
# [Zeppelin 796] Duplicated notebook names should not be allowed

What is this PR for?

When a notebook is created/cloned/imported the title/name of the new notebook should not be duplicate of the existing notebook's title/name

What type of PR is it?

Improvement

Todos

 - Task
What is the Jira issue?

https://issues.apache.org/jira/browse/ZEPPELIN-796

How should this be tested?

Create a notebook with name 'Note', Notebook should allow user to create a note
When a user try to create/clone/import a notebook with name 'Note', Notebook should not allow user to create note.
Screenshots (if appropriate)

Screenshot of error message displayed during create/clone notebook
![create](https://cloud.githubusercontent.com/assets/11922701/19467185/725d9dce-952d-11e6-8bee-bd1b19397993.png)


Screenshot of error message displayed during import notebook
![import](https://cloud.githubusercontent.com/assets/11922701/19467194/7cdc9b9c-952d-11e6-84e4-c9edcf37a4ef.png)


Questions:

Does the licenses files need update? No
Is there breaking changes for older versions? No
Does this needs documentation? No
